internal/pkg/describe/webapp.go

```diff
@@ -205,6 +198,25 @@ func (d *WebAppDescriber) ECSParams(envName string) (*WebAppECSParams, error) {
 	}, nil
 }
 
+// GetServiceArn returns the ECS service ARN of the application in an environment.
+func (d *WebAppDescriber) GetServiceArn(envName string) (*ServiceArn, error) {
+	env, err := d.store.GetEnvironment(d.app.Project, envName)
+	if err != nil {
+		return nil, err
+	}
+	appResources, err := d.describeStackResources(env.ManagerRoleARN, env.Region, stack.NameForApp(d.app.Project, env.Name, d.app.Name))
+	if err != nil {
+		return nil, err
+	}
+	for _, appResource := range appResources {
+		if aws.StringValue(appResource.LogicalResourceId) == serviceLogicalID {
+			serviceArn := ServiceArn(aws.StringValue(appResource.PhysicalResourceId))
+			return &serviceArn, nil
```

Comment:
nit: return the value instead of the pointer, since the receivers are values now